plugins/npm/test/npm-publish.test.ts

```diff
@@ -69,7 +69,7 @@ describe('NpmPublish', () => {
     await expect(async () => {
       await task.run()
     }).rejects.toThrowErrorMatchingInlineSnapshot(
-      `"The Tool Kit \`ci\` state \`tag\` variable random-branch does not match regex /^v\\\\d+\\\\.\\\\d+\\\\.\\\\d+(-.+)?/. Configure your release version to match the regex eg. v1.2.3-beta.8"`
+      `"The Tool Kit \`ci\` state \`tag\` variable random-branch does not match regex /^v\\d+\\.\\d+\\.\\d+(-.+)?/. Configure your release version to match the regex eg. v1.2.3-beta.8"`
```

Comment:
**thought**:

![image](https://github.com/Financial-Times/dotcom-tool-kit/assets/631757/fd81d8ec-e733-43cc-b4f6-65a62a49ede7)

Reply:
wow an xkcd i've never seen before. probably because it's not that funny

Reply:
it's _old_